js/pronoun_text.js

```diff
@@ -0,0 +1,29 @@
+
+function initAddPronounText() {
+  var windowUrl = window.location.pathname;
+  var onProfileSettingsPage = (windowUrl.search(/\/profile\/settings/) != -1);
+
+  if (onProfileSettingsPage) {
+    textSpan = $( "#pronouns" ).parent().children().last();
+
+    textSpan.html("\
+    <br/>This pronoun will appear after your name when enabled.\
+    If you would like to do so, use this section to identify, \
+    change, or delete your chosen personal pronouns. Currently \
+    Canvas requires users to choose from a limited \
+    prepopulated list of personal pronouns. We respect that \
+    language is constantly evolving and members of our community \
+    should be able to be called and describe their identity \
+    however they choose, so we are working to enable that option \
+    within Canvas. To learn more about gender, identity, \
+    or pronouns and to provide feedback, visit: \
+    <a href=\"https://dib.harvard.edu/gender-pronouns\">\
+    https://dib.harvard.edu/gender-pronouns</a>.\
+    ");
+
+    $( "#pronouns" ).parent().parent().children("th").css("vertical-align", "top");
+
+  }
+}
+
+$(document).ready(initAddPronounText);
```

Comment:
Q: is the [branding page I found on k459](https://wiki.harvard.edu/confluence/pages/viewpage.action?pageId=157484597) up-to-date, per deploying these? Also, should the global JS files be part of the code committed, per the [README](https://github.com/harvard-canvas-branding/canvas-branding-global#readme)?

Reply:
The page on k459 doesn't really talk about deployment, but the readme should be updated.